src/editor.py

```diff
@@ -194,7 +194,7 @@ def generate_header(self,files):
             timezone = self.time_zone
             date_object = date_object - t
             header = "**First Pitch:** " + date_object.strftime("%I:%M %p ") + timezone + "\n\n"
-            header = header + "[Preview](http://mlb.mlb.com/mlb/gameday/index.jsp?gid=" + game.get('gameday_link') + ")\n"
+            header = header + "[Preview](http://mlb.mlb.com/mlb/gameday/index.jsp?gid=" + game.get('gameday_link') + ")\n\n"

```

Comment:
If the GDT is posted a few hours before game time, it is posted with only the Preview link and now the footer. Without this change, the footer text shows on the same line as the Preview link when the rest of the header text is not yet available.